PeerioTechnologies/peerio-desktop
# fix: open active patient space on login and return to chats from elsewhere

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/8951/mc-stay-in-patient-space-when-switching-tabs
 
#### Testing instructions  
In MC, when (a) patient space is open with no active chat, or (b) a Patient Room or Internal Room is open:
* active patient space/room should show on login
* patient sidebar should show when navigating away from chats and then back again

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
